fake/fake-request-service.js

```diff
@@ -279,9 +285,9 @@ var details = (function () {
             }
         }
     });
-    
+
     // stream subscribers
-    streamMock.on(window.location.origin + '/glimpse/message-stream/', function() {
+    streamMock.on(window.location.origin + '/glimpse/message-stream/', function () {
         summaries.stream();
     });
 })();
```

Comment:
There are a fair few white space changes here. Are they all needed for this PR (specifically `function ()` vs `function()` - in most cases we use the later)?